pkg/await/await.go

```diff
@@ -116,7 +118,7 @@ func Creation(c CreateConfig) (*unstructured.Unstructured, error) {
 				return err
 			}
 
-			return err
+			return nil
```

Comment:
Recapping offline discussion, we've agreed to take this change out because, though small, it is _very_ important that this PR in particular does not add bugs. :)